bool enable_platooning
float32 inner_platoon_distance
float32 platoon_speed